ui/ui.go

```diff
@@ -185,9 +201,24 @@ func newPrdHandler(cfg *config.Config) (http.Handler, error) {
 			return nil, err
 		}
 
-		cnt.tpl, err = template.New(cnt.template).Parse(string(a))
-		if err != nil {
-			return nil, err
+		// For more context, see: https://github.com/etsy/hound/issues/239
```

Comment:
Dry this bit up into a function :)

Reply:
It's actually different for prod and dev.

Reply:
In prod, we load the contents of template from bindata, however, in dev mode, we load the contents of template directly from the file.